referencia/webpayrest/README.md

```diff
@@ -1053,22 +1192,25 @@ capturedAmount  <br> <i> Decimal </i> | Monto que se desea capturar. Largo má
 200 OK
 Content-Type: application/json
 {
-  "token": "e074d38c628122c63e5c0986368ece22974d6fee1440617d85873b7b4efa48a3",
+  "type": "NULLIFY",
   "authorization_code": "123456",
   "authorization_date": "2019-03-20T20:18:20Z",
-  "captured_amount": 1000
+  "nullified_amount": 1000.00,
+  "balance": 0.00,
+  "response_code": 0
 }
 ```
 
 Nombre  <br> <i> tipo </i> | Descripción
 ------   | -----------
-token  <br> <i> String </i> | Token de la transacción. Largo máximo: 64
-authorizationCode  <br> <i> String </i> | Código de autorización de la captura diferida. Largo máximo: 6
-authorizationDate  <br> <i> String </i> | Fecha y hora de la autorización.
-capturedAmount  <br> <i> Decimal </i> | Monto capturado. Largo máximo: 6
+type  <br> <i> String </i> | Tipo de reembolso (REVERSE. NULLIFY). Largo máximo: 10
+authorization_code  <br> <i> String </i> | Código de autorización de la anulación. Largo máximo: 6
+authorization_date  <br> <i> String </i> | Fecha y hora de la autorización.
+balance  <br> <i> Decimal </i> | Saldo actualizado de la transacción (considera la venta menos el monto anulado). Largo máximo: 17
+nullified_amount  <br> <i> Decimal </i> | Monto anulado. Largo máximo: 17
+response_code <br> <i> Number </i> | Código de resultado del pago. Si es exitoso es 0, de lo contrario el pago no fue realizado. Largo máximo: 2
```

Comment:
again, this should say `anulación` or `reembolso`

Reply:
Fixed in d09e83d